tests/view_queries_test.py

```diff
@@ -103,3 +103,19 @@ def test_should_create_enriched_offerer_data_view_with_columns(self, app):
             offerers_details = pandas.read_sql_table('enriched_offerer_data', CONNECTION, index_col='offerer_id')
             for expected_column_in_view in expected_columns:
                 assert expected_column_in_view in offerers_details.columns
+
+    class CreateBookableOffersTest:
+        def test_should_create_all_bookable_offers_view(self, app):
+            # when
+            with app.app_context():
+                create_all_bookable_offers_view()
+
+            # then
+            expected_columns = ["idAtProviders", "dateModifiedAtLastProvider", "dateCreated", "productId", 
+            "venueId", "lastProviderId", "bookingEmail", "isActive", "type", "name", "description", "conditions", "ageMin",
+            "ageMax", "url", "mediaUrls", "durationMinutes", "isNational", "extraData", "isDuo", "fieldsUpdated"]
+
+            bookable_offers_df = pandas.read_sql_table('all_bookable_offers', CONNECTION, index_col='id')
+            for expected_column_in_view in expected_columns:
+                assert expected_column_in_view in bookable_offers_df.columns
```

Comment:
Ici on n'a pas besoin de faire une boucle, on peut faire directement
```suggestion
                assert sorted(expected_columns) == sorted(bookable_offers_df.columns)
```

Reply:
smart !